docs/modules/ROOT/pages/api/token_common.adoc

```diff
@@ -0,0 +1,192 @@
+:github-icon: pass:[<svg class="icon"><use href="#github-icon"/></svg>]
+:eip2981: https://eips.ethereum.org/EIPS/eip-2981[EIP-2981]
+
+= Common (Token)
+
+include::../utils/_common.adoc[]
+
+Reference of extension and utilities that are common to multiple token standards.
+
+== ERC2981
+
+[.contract]
+[[IERC2981]]
+=== `++IERC2981++` link:https://github.com/OpenZeppelin/cairo-contracts/blob/release-v0.15.0/packages/token/src/common/erc2981/interface.cairo[{github-icon},role=heading-link]
+
+[.hljs-theme-dark]
+```cairo
+use openzeppelin::token::common::erc2981::IERC2981;
+```
```

Comment:
Sorry! I'm just noticing this. We should probably include the SRC5 block for the interface